hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-pt.tex

```diff
@@ -1,58 +1,47 @@
+%           of Minho nor the names of its contributors may be used to endorse
+%           or promote products derived from this software without specific
+%           prior written permission.
 % 
-% THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
-% ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-% WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-% DISCLAIMED. IN NO EVENT SHALL PEDRO J. DE REZENDE OR J.JOAO DIAS ALMEIDA BE
-% LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-% CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE
-% GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
-% HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
-% LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT
-% OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-% The Portuguese TeX hyphenation table.
-% (C) 2015 by  Pedro J. de Rezende (rezende@ic.unicamp.br)
-%          and J.Joao Dias Almeida (jj@di.uminho.pt)
-% Version: 1.3 Release date: 12/08/2015
-%
-% (C) 1996 by  Pedro J. de Rezende (rezende@ic.unicamp.br)
-%          and J.Joao Dias Almeida (jj@di.uminho.pt)
-% Version: 1.2 Release date: 07/21/1996
-%
-% (C) 1994 by Pedro J. de Rezende (rezende@ic.unicamp.br)
-% Version: 1.1 Release date: 04/12/1994
-%
-% (C) 1987 by Pedro J. de Rezende
-% Version: 1.0 Release date: 02/13/1987
-%
-% -----------------------------------------------------------------
-% Remember! If you *must* change it, then call the resulting file
-% something  else and attach your name to your *documented* changes.
-% =================================================================
+%         THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+%         "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+%         LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+%         A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL PEDRO J. DE
+%         REZENDE OR J.JOAO DIAS ALMEIDA BE LIABLE FOR ANY DIRECT, INDIRECT,
+%         INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+%         BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+%         OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+%         AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+%         LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+%         WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+%         POSSIBILITY OF SUCH DAMAGE.
+% changes:
+%     - Version 1.3 Release date: 12/08/2015
+%     - Version 1.2 Release date: 07/21/1996
+%     - Version 1.1 Release date: 04/12/1994
+%     - Version 1.0 Release date: 02/13/1987
```

Comment:
I don't like dropping the details of who contributed what.

Reply:
Reinstated.